reference/clever-tools/networkgroups.md

```diff
@@ -0,0 +1,192 @@
+---
+title: How to use `clever networkgroups`
+shortdesc: Configure networkgroups using the Clever Cloud CLI tool
+tags:
+- cli
+- reference
+- networkgroups
+keywords:
+- cli
+- clever-tools
+- join
+- wireguard
+- network
+- vpn
+- client
+- server
+---
+
+## Overview
+
+> TODO: Describe networkgroups
+
+## How to create & join networkgroups with the `clever` CLI?
+
+Under each command, I added an example output for you to see what you're supposed to get.
```

Comment:
There should not be a narrator speaking in first person.

A better alternative would be: "An example is provided under each command to see the expected result".